docs/ProblemStatementAndGoals/ProblemStatement.tex

```diff
@@ -40,8 +41,27 @@ \subsection{Problem}
 
 \subsection{Inputs and Outputs}
 
-\wss{Characterize the problem in terms of ``high level'' inputs and outputs.  
-Use abstraction so that you can avoid details.}
+\subsubsection{Inputs}
+    \begin{itemize}
+        \item Personal consumer spending data
+        \item Location data
+       %\item User Preferences
+        \item Historical Shopping Lists
+        %\item User Reviews and Ratings
+        \item Shopping History
+    \end{itemize}
+
+\subsubsection{Outputs}
+    \begin{itemize}
+        \item Analysis of spending data
+        \item Purchase suggestions based on spending data
+        \item Budgeting Plans
+        \item Local Price Comparison
+        \item Shopping Trip Summaries
+        %\item Community Forums
```

Comment:
can we remove these comments? commenting out code is useful when you are in the midst of working ... try to avoid committing these though ... this is why we have VC to look back on our history. So there is no need to keep this in the head.

Reply:
resolved: removed comments